githooks.sh

```diff
@@ -919,76 +946,76 @@ update_release_clone() {
     fi
 
     if [ "$PULL_ONLY" = "true" ]; then
-        PULL_OUTPUT=$(
-            git -C "$CLONE_DIR" \
-                --work-tree="$CLONE_DIR" \
-                --git-dir="$CLONE_DIR/.git" \
-                -c core.hooksPath=/dev/null pull origin "$GITHOOKS_CLONE_BRANCH" 2>&1
+
+        FETCH_OUTPUT=$(
+            execute_git "$GITHOOKS_CLONE_DIR" remote update origin 2>&1
         )
 
         # shellcheck disable=SC2181
         if [ $? -ne 0 ]; then
-            echo "! Pulling updates in  \`$CLONE_DIR\` failed with:" >&2
-            echo "$PULL_OUTPUT" >&2
+            echo "! Fetching updates in  \`$GITHOOKS_CLONE_DIR\` failed with:" >&2
+            echo "$FETCH_OUTPUT" >&2
             return 1
         fi
+
+        GITHOOKS_RELEASE_COMMIT=$(execute_git "$GITHOOKS_CLONE_DIR" rev-parse "$GITHOOKS_CLONE_BRANCH")
```

Comment:
no explizit is better here. Refactored ths update shizzle... in my next commits

Reply:
But is using the branch name more explicit? It seems like a roundabout way to express what you mean (which is, I think, is the *currently checked out version*, which is HEAD, different from the fetched version). But if you refactored this, this might no longer be relevant :-)